src/common/TableOfContents.ts

```diff
@@ -0,0 +1,12 @@
+export interface HeadingLink {
+  content: string
+  anchorLink: string
+  level: number
+}
+
+export interface TableOfContentsProps {
+  type: "tableofcontents"
+  headings: HeadingLink[]
```

Comment:
nit: maybe we can name this as `items` instead?

Reply:
[bff3d13](https://github.com/isomerpages/isomer-components/pull/34/commits/bff3d13e9d5f1f30f3e4bd132777a7626b5313a1)